src/file-selector.ts

```diff
@@ -23,11 +20,6 @@ export async function fromEvent(
     return getDataTransferFiles(evt.dataTransfer, evt.type);
   } else if (isChangeEvt(evt)) {
     return getInputFiles(evt);
-  } else if (
-    Array.isArray(evt) &&
-    evt.every((item) => "getFile" in item && typeof item.getFile === "function")
-  ) {
-    return getFsHandleFiles(evt);
```

Comment:
This would be a breaking change. I imagine it might affect quite a few downstream clients.

It would be good if the commit specifically mentions that (it's currently not explicit that `fromEvent` doesn't handle lists of file handles).

This will break the default behaviour of react-dropzone as well since it uses `fromEvent` as the default `getFilesFromEvent` .

Reply:
It might make sense to keep this func as it was and call the new func here to avoid breaking the behaviour of react-dropzone.

Reply:
Since we're already shipping several breaking changes on the `beta` branch I figure this is the best time to make any API changes. 

> It would be good if the commit specifically mentions that (it's currently not explicit that `fromEvent` doesn't handle lists of file handles).

Sure, how would you like me to do so? I can add this to the `BREAKING CHANGE:` section of the commit description, that should add it to the release notes. Additionally, I intend to write a migration guide for users after the dust has settled on the `beta` branch, which will likely be a little markdown document linked from the README.

WDYT?

Reply:
Sounds good.